File: node.js/philippkueng.node-neo4j/test/example-test.js
var example = require("../example")
var assert = require("assert")

describe('helloCypher', function(){
  it('should get my friends', function(done){
    example.helloCypher(function(friends) {
      assert.equal(friends[0], "JnBrymn");
      assert.equal(friends[1], "technige");
      done();
    });
  })
})

File: scala/anormcypher/src/test/scala/exampletest.scala
package anormcypher.example

import org.scalatest._

class HelloCypherSpec  extends FlatSpec with Matchers {
  "HelloCypher" should "be able to run a cypher query" in {
    Example.helloCypher.toList should be (List("JnBrymn", "technige"))
  } 
}
